gensim/test/test_fasttext.py

```diff
@@ -1114,6 +1114,29 @@ def test_save_load_native(self):
 
             model.save(model_name)
 
+    def test_load_native_pretrained(self):
+        model = FT_gensim.load_fasttext_format(datapath('toy-model-pretrained.bin'))
+        actual = model['monarchist']
+        expected = np.array([0.76222, 1.0669, 0.7055, -0.090969, -0.53508])
+        self.assertTrue(np.allclose(expected, actual, atol=10e-4))
+
+
+def _train_model_with_pretrained_vectors():
+    """Generates toy-model-pretrained.bin for use in test_load_native_pretrained.
```

Comment:
PEP257: docstrings in imperative mode (`Generates …` => `Generate …`).

Reply:
Done